src/Main.java

```diff
@@ -0,0 +1,11 @@
+public class Main{
+    public static void main(String[] args) {
+        Game game = new Game();
+        game.database.addWord("drzewo");
```

Comment:
Add words to database when initializing database or in Game contructor. Let's only have game.hangman() in main method.